doc/OCP_LOCK.md

```diff
@@ -0,0 +1,256 @@
+![LOCK Logo](./images/lock/LOCK_logo_large.png)
+
+# OCP L.O.C.K.
+
+OCP L.O.C.K. (Layered Open-source Cryptographic Key management) is a feature set conditionally compiled into Caliptra, which provides secure key management for data-at-rest protection in self-encrypting storage devices.
```

Comment:
Worth clarifying, it is conditionally compiled in FW, but the hardware is the same (does not require synthesizing with a different config). Is that correct?

Reply:
We don't believe any hardware will change with L.O.C.K., but we haven't gotten into the design in sufficient detail to assure ourselves that zero HW changes will be needed when synthesizing.